bot/cogs/grades_data/grades_cog.py

```diff
@@ -234,6 +242,59 @@ def searchCourse(self, query, year):
     def go(self, course, year):
         return self.searchCourse(course, year)
 
+    def get_professor_query(self, prof_name, detailed):
+        # print("NAME: " + prof_name)
+        primary = self.global_master_prof_list[prof_name]
+        courses = {}
+        
+        for section in primary:
+            course = section['course'] + "-" + section['number']
```

Comment:
realy bink -_-

Reply:
Fine I'll fix it. F strings. This logic I understand. Your other statement last night, I did not.

Reply:
the list comp tuple?

Reply:
The whole "Make it a set" idea you had and code you sent over. I had no idea what it was doing so I've elected to ignore it